src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -198,6 +207,26 @@ library BeaconChainProofs {
         require(Merkle.verifyInclusionSha256(proof, beaconStateRoot, balanceRoot, balanceIndex), "BeaconChainProofs.verifyValidatorBalance: Invalid merkle proof");
     }
 
+    function verifySlotRoot(
+        bytes32 beaconStateRoot,
+        bytes calldata proof,
+        bytes32 slotRoot
+    ) internal view {
+        require(proof.length == 32 * (BEACON_STATE_FIELD_TREE_HEIGHT), "BeaconChainProofs.verifySlotRoot: Proof has incorrect length");
+        //Next we verify the slot against the blockHeaderRoot
+        require(Merkle.verifyInclusionSha256(proof, beaconStateRoot, slotRoot, BEACON_STATE_SLOT_INDEX), "BeaconChainProofs.verifyWithdrawalProofs: Invalid slot merkle proof");
+    }
+
+    function verifyStateRootAgainstLatestBlockHeaderRoot(
```

Comment:
would recommend we agree on the order like
`root, leaf, proof`

Reply:
for parameters into `verify...(` functions

Reply:
I agree a standardized order would be really nice.
this function is also missing NatSpec documentation